kata-deploy/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos/systemd
+ARG KATA_VER=1.0.0
+ARG KATA_URL=https://github.com/kata-containers/runtime/releases/download/${KATA_VER}
+
+RUN yum install -y wget
+WORKDIR /tmp/kata/
+RUN wget -q ${KATA_URL}/{vmlinuz.container,kata-containers.img}
+
+WORKDIR /tmp/kata/bin/
+RUN wget -q ${KATA_URL}/{kata-runtime,kata-proxy,kata-shim}
```

Comment:
Can combine the layers into one like:
ARG KUBECTL_VER=v1.10.2

RUN \
  curl -s -o kata-runtime ${KATA_URL}/kata-runtime && \
  curl -s -o kata-proxy ${KATA_URL}/kata-proxy && \
  curl -s -o kata-shim ${KATA_URL}/kata-shim && \
  curl -s -o /bin/kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VER}/bin/linux/amd64/kubectl && \
  chmod +x /bin/kubectl